components/dashboard/src/App.tsx

```diff
@@ -183,14 +181,6 @@ function App() {
                     }
                 }
                 setTeams(teams);
-
-                if (user?.rolesOrPermissions?.includes("admin")) {
-                    const adminSettings = await getGitpodService().server.adminGetSettings();
-                    setAdminSettings(adminSettings);
-
-                    var license: LicenseInfo = await getGitpodService().server.adminGetLicense();
-                    setLicense(license);
```

Comment:
We never used those? :exploding_head:

Reply:
Potential problem: https://werft.gitpod-dev.com/job/gitpod-build-at-admingetlicense.0#build:components/dashboard:app

Reply:
They are, but in Settings.tsx/License.tsx, where there is another call already. 

Thanks for the pointer to the build issue. Fixed.